src/components/common/flex.tsx

```
@@ -0,0 +1,83 @@
+        $width={width}
+        $height={height}
+        $gap={gap}
+        $margin={margin}
+        $overflow={overflow}
+        {...props}
+      >
+        {children}
+      </StyledFlex>
+    );
+  }
+);
```

ESLint

[error] Component definition is missing display name

(react/display-name)